lib/screenplay_web/router.ex

```diff
@@ -63,6 +67,17 @@ defmodule ScreenplayWeb.Router do
     get("/unauthorized", UnauthorizedController, :index)
   end
 
+  scope "/", ScreenplayWeb do
+    pipe_through([
+      :redirect_prod_http,
+      :browser,
+      :auth,
+      :ensure_auth,
+      :ensure_pa_message_admin,
```

Comment:
`pipe_through` accepts plug modules as well as pipelines; we technically don't need to define a pipeline if it only consists of one module. (Though maybe this could be better for readability?)

Reply:
Hmm, I could go either way on this. A part of me does feel like a list of only atoms is more readable than a list of atoms and module names. Unless you feel strongly one way or the other, I think I'll keep it as-is.